src/odf/metadata/src/engine/grpc_generated/engine.rs

```diff
@@ -7,23 +7,30 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+// Generated with protoc-gen-tonic=0.4.1
+// Generated with protoc-gen-prost=0.4.0
+// Generated with protoc=libprotoc 29.3
```

Comment:
Note: adding toolset versions to generated files
In future we should containerize them to make codegen reproducible and not depend on the local env.